backend/amd64/simd.ml

```diff
@@ -30,6 +30,13 @@ type float_condition = X86_ast.float_condition =
   | NLEf
   | ORDf
 
+type float_rounding = X86_ast.rounding =
```

Comment:
it doesn't need to be redefined, but I think it's better to use `X86_ast.rounding` since the values get directly forwarded to the x86 emitters, which expect that type